tests/test_redis_connection.py

```diff
@@ -0,0 +1,12 @@
+from typing import Final
+
+import pytest
+from redis import Redis
+
+REDIS_HOST: Final = "redis"
+
```

Comment:
Note in a true local environment separate from docker, this would have to be "0.0.0.0"